deployment/data-streams/changeset/testutil/token.go

```diff
@@ -0,0 +1,56 @@
+package testutil
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commonstate "github.com/smartcontractkit/chainlink/deployment/common/changeset/state"
+)
+
+func MustFundAddressWithLink(t *testing.T, e deployment.Environment, chain deployment.Chain, to common.Address, amount int64) {
```

Comment:
I think we could move this to common in the future as other teams can benefit of this as well.